crates/utils/src/brush.rs

```diff
@@ -1,67 +1,76 @@
-/// a color as `hsl(197, 71%, 73%)` and you will get a pretty skyblue color. For `rgb` and `rgba` the
-/// range of the values are 0-255 but from all others the values are as follows: hsva(0.0-360.0, 0.0-1.0, 0.0-1.0, 0.0-1.0)
-/// but the % sign can change that, it can be added to individual parameters and change its range depending on the function,
-/// for `rgb` and `rgba` you can write an % after a parameter and it will be mapped from 0.0-100.0, or if the number that you write
-/// is between 0.0 and 1.0 you value will be mapped from that range. But if you function is not rgb or rgba, you can not use a
-/// percent sign in the first parameters but in the following parameters the sign will make OrbTk interpret it in the 0.0-100.0 range.
+/// Define the value with a symbol "#" followed by 6 letters or numbers.
+/// These numbers are in hexadecimal numeral system.
+/// For example `#f00` will give you red. If you write `#00ff00`, you will
+/// get blue and you also included an alpha channel.
 ///
-/// C. Use the color name! so you want to write a colour and do you code work lazily?, no problem! you can directly write the name.
-/// For example white, red, olive, ivory, linen or any of the CSS color names!, also the next extra colors are supported:
-/// * alabaster
-/// * alto
-/// * bluebayoux
-/// * bombay
-/// * brightgrey
-/// * dolly
-/// * energyellow
-/// * fiord
-/// * goldendream
-/// * goldenfizz
-/// * goldtips
-/// * gorduroy
-/// * governorbay
-/// * linkwater
-/// * lynch
-/// * manatee
-/// * manz
-/// * mineshaft
-/// * periwinklegrey
-/// * portage
-/// * sandwisp
-/// * silverchalice
-/// * sunflower
-/// You also can define a gradient with an expression, the syntax is as follows:
+/// B. Use `a function`
+///
+/// Currently the unique available functions that interprete a color are
+/// destincted with the keywords `rgb`, `hsv`, `hsb`, `hsl`. There are
+/// `alpha variants` as well. `hsb` is an alias to `hsv`.
+/// Alpha variants are coded with the keywords `rgba`, `abgr`  or `argb`.
+/// Here is an example to define a color via the function method:
+/// `hsl(197, 71%, 73%)` will provide you a pretty skyblue color.
+/// For `rgb` and `rgba` the range of the values are 0-255.
+/// Any other keyword will use floating point integers to define the color
+/// value. `hsva(0.0-360.0, 0.0-1.0, 0.0-1.0, 0.0-1.0)` is such an example.
+/// In addition you can choose to use percent values (`%` sign) for the given
+/// parameters.
+/// When appending the `%` sign to the range parameters of the `rgb` function
+/// call, the values are mapped to 0.0-100.0 (percent) or 0.0-1.0 (min/max).
+/// For all other keywords (`hsv`, `hsb`, `hsl`) you are not allowed to append
+/// the percent sign to the first parameter. If you append `%` to the following
+/// parameters, OrbTk will interpret the values in a range between `0.0-100.0`.
+///
+/// C. Use the `color name`
+///
+/// OrbTK maintains a color name map (crates/utils/colors.txt). It enables
```

Comment:
Yes you can do a link to the original but that forces the user to look at the source code and do an extra step, I don't think extensive documentation is a problem(I always use the index to navigate the docs like most people do)